packages/vm/core/evm/evmtest/contractInstance.go

```diff
@@ -0,0 +1,166 @@
+package evmtest
+
+import (
+	"crypto/ecdsa"
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/stretchr/testify/require"
+
+	"github.com/iotaledger/wasp/packages/isc"
+)
+
+type EVMContractInstance struct {
+	chain         *SoloChainEnv
+	defaultSender *ecdsa.PrivateKey
+	address       common.Address
+	abi           abi.ABI
+}
+
+type CallFnResult struct {
+	tx         *types.Transaction
+	EvmReceipt *types.Receipt
```

Comment:
Nit: following Go naming conventions [*](https://github.com/golang/go/wiki/CodeReviewComments#initialisms):

```suggestion
	EVMReceipt *types.Receipt
```

Also nit: kinda arbitrary that `EVMReceipt` is public but `iscReceipt` is not. Same for the other structs made public.

Reply:
Good point 👍